.buildkite/pipeline.yml

```diff
@@ -113,6 +113,13 @@ steps:
           slurm_ntasks: 2
           slurm_mem: 16GB
 
+      - label: "GPU Utilities unit tests"
```

Comment:
This looks good! We also want to add an MPI test to cover the 3rd comms_ctx case. You can add another MPI unit test that follows the structure of the tests above here but is for utilities_tests.jl